Template10 (Library)/Services/NavigationService/NavigationService.cs

```diff
@@ -268,7 +268,11 @@ public void Navigate<T>(T key, object parameter = null, NavigationTransitionInfo
         public ISerializationService SerializationService { get; set; }
 
         public event EventHandler<CancelEventArgs<Type>> BeforeSavingNavigation;
-        public async Task SaveNavigationAsync()
+
+        
+
+
+        public async Task SaveAsync()
```

Comment:
There may very well be developers using SaveSavedNavigationAsync and so we won't rename it. Instead, mark it with [Obsolete] and create a duplicate called SaveAsync(). Okay?